packages/2019-disaster-game/src/components/Game/TaskScreen/mapUtils.js

```diff
@@ -74,7 +74,7 @@ export const getPosition = f =>
 export const asGeoJSON = (tasks, activeTask, completedTasks) =>
```

Comment:
maybe rename this function as it seems to be doing a lot more than asGeoJSON

Reply:
Since I didn't need to implement this function and would need to dig in to understand it, I'm going to pass on this for now in lieu of more important tasks